sdk/appconfiguration/app-configuration/test/internal/http.spec.ts

```diff
@@ -103,7 +108,9 @@ describe("http request related tests", function() {
 
     beforeEach(function() {
       recorder = startRecorder(this);
-      client = createAppConfigurationClientForTests() || this.skip();
+      client =
+        createAppConfigurationClientForTests<InternalAppConfigurationClientOptions>({}) ||
+        this.skip();
```

Comment:
Is this change really needed?

Reply:
That happened as a result of formatting changes by git hooks - will undo that